pom.xml

```diff
@@ -405,6 +367,15 @@
     </scm>
 
     <profiles>
+        <profile>
+            <id>java9-release8</id>
+            <activation>
+                <jdk>[9,)</jdk>
+            </activation>
+            <properties>
+                <maven.compiler.release>8</maven.compiler.release>
```

Comment:
won't this be defined in the parent pom?

Reply:
It may have also obscured deprecation issues in Java 17+